ql/src/java/org/apache/hadoop/hive/ql/stats/ColumnStatisticsObjTranslator.java

```diff
@@ -59,7 +59,7 @@ public static ColumnStatisticsObj readHiveColumnStatistics(String columnName, St
       List<ColumnStatsField> columnStatsFields, int start, List<? extends StructField> fields,
       List<Object> values) throws HiveException {
     ColumnStatisticsObj statsObj = new ColumnStatisticsObj();
-    statsObj.setColName(columnName);
+    statsObj.setColName(columnName.toLowerCase());
```

Comment:
How this fix the lower case? Somewhere is a compare the column name and in other places could be capital?

Reply:
it's always in lowercase in Hive schema, but not in ColumnStatisticsObj